src/core/frames/frame_controller.ts

```diff
@@ -247,6 +247,10 @@ export class FrameController
 
   requestErrored(request: FetchRequest, error: Error) {
     console.error(error)
+    dispatch<TurboFetchErrorEvent>("turbo:fetch-error", {
```

Comment:
How do you feel about naming the event `turbo:fetch-request-error`? Other [fetch-related events](https://turbo.hotwired.dev/reference/events) are named to describe which side of they pipe they're dispatched on: `turbo:before-fetch-request` and `turbo:before-fetch-response`.

Reply:
Also, could you add the type to the [exports from `src/core/index.ts`](https://github.com/hotwired/turbo/blob/60c646383b3c0355242d2e3c69f80742efb266be/src/core/index.ts#L17-L31)?

Reply:
Done in db678ec176d74eff9c82876e05b979b1fddc16c2! I'm still fighting tests a bit.